destalinator.py

```diff
@@ -34,7 +38,14 @@ def __init__(self, slacker, slackbot, activated):
         print("output_debug_to_slack_flag is {}".format(self.output_debug_to_slack_flag))
         self.destalinator_activated = activated
         print("destalinator_activated is {}".format(self.destalinator_activated))
-        self.earliest_archive_date = self.config.earliest_archive_date
+
+        archive_date_string = os.getenv(self.config.earliest_archive_date_env_varname)
```

Comment:
`getenv` takes a default as the second parameter, moving "2000-01-01" there would clean this up quite a bit.